site/documentation/content/deployment/helm-based-deployment.md

```diff
@@ -186,7 +186,7 @@ Once the images have been pushed, the deployment can be done using Helm:
 
 ~~~sh
 # in Hono working tree directory: hono/deploy
-helm install --dep-up --name hono --namespace hono --set honoContainerRegistry=my.registry.io target/deploy/helm/
+helm install --dep-up --name hono --namespace hono target/deploy/helm/eclipse-hono/
```

Comment:
FMPOV this will only work if not using a custom tag when pushing the images to the private container registry. Otherwise, the images will not be found when the pods are starting up because they are looked up using the Maven `${project.version}` instead of the custom tag ...

Reply:
That is independent of the registry setting, right?

But true I could add a chapter on the custom tag topic.

Reply:
Ah, now I get it. The push images line in docs defines "1.0-CUSTOM"

Reply:
BTW, there are still merge conflicts ...

Reply:
@sophokles73 are you sure, I merged today with master and GitHub tells me no conflicts? I updated the docs btw.

Reply:
> This branch cannot be rebased due to conflicts
Rebasing the commits of this branch on top of the base branch cannot be performed automatically due to conflicts encountered while reapplying the individual commits from the head branch.